examples/raspberry-pi/facial-recognizer-rPi.json

```diff
@@ -0,0 +1,241 @@
+[
+  {
+      "id": "e5f70c65.350d1",
+      "type": "tab",
+      "label": "Facial Recognizer",
+      "disabled": false,
+      "info": ""
```

Comment:
Can you add info/help similar to what we already have for the getting started flows to this flow?